packages/salesforcedx-vscode-core/test/vscode-integration/telemetry/index.test.ts

```diff
@@ -177,267 +175,5 @@ describe('Telemetry', () => {
       assert.notCalled(mShowInformation);
       expect(teleStub.firstCall.args).to.eql([true]);
     });
```

Comment:
So all these tests have been x'd out forever. Didn't see a good reason to keep them around esp considering the tests in this file are kind of bonkers.